kubeflow/profiles/prototypes/README.md

```diff
@@ -0,0 +1,97 @@
+## Overview
+
+### Goals
```

Comment:
Can you update the README.md to make it clear what an admin would do vs. what each user would do to create their own profile?

I'm a little confused because the SecurityProfile doc 
https://github.com/kubernetes/community/blob/a8cb2060dc621664c86b185c7426367994b181b5/keps/draft-20180418-security-profile.md

Makes it seem like you define a SecurityProfile and then you create resources according that profile.

So if we are following that design, I would expected there to be a SecurityProfile that would be defined by the admin and then individual resources (e.g. namespaces) would be created with that Profile.

In the SecurityProfile doc for example, the Profile seems to describe the security policy for a cluster and then multiple clusters are created with that profile.

Is the reason we have to define a different SecurityProfile for each user, is because each SecurityProfile needs to correspond to a different set of users to grant access to?

Reply:
There are rolebindings created for each user in the new namespace. 
A user can have multiple namespaces (by submitting multiple Profiles), 
but each namespace contains a Role, Rolebinding.
All namespaces reference the user's ServiceAccount in kubeflow.

So if fred submits 2 profiles: resnet50, inceptionv3, there would 2 namespaces, each one having a {Role, RoleBinding}.
```
      apiVersion: "rbac.authorization.k8s.io/v1",
      kind: "RoleBinding",
      metadata: {
        name: "fred",
        namespace: "resnet50",
      },
      roleRef: {
        apiGroup: "rbac.authorization.k8s.io",
        kind: "Role",
        name: "edit",
      },
      subjects: [{
        kind: "ServiceAccount",
        name: "fred",
        namespace: "kubeflow",
      }
```
```
      apiVersion: "rbac.authorization.k8s.io/v1",
      kind: "RoleBinding",
      metadata: {
        name: "fred",
        namespace: "inceptionv3",
      },
      roleRef: {
        apiGroup: "rbac.authorization.k8s.io",
        kind: "Role",
        name: "edit",
      },
      subjects: [{
        kind: "ServiceAccount",
        name: "fred",
        namespace: "kubeflow",
      }
```

The 'edit' Role is the same for all namespaces created.
The RoleBinding just differs by the user's name.

The RoleBinding could be parameterized the type of Role

Reply:
> Can you update the README.md to make it clear what an admin would do vs. what each user would do to create their own profile?

Yes, will do